checkstyle/suppressions.xml

```diff
@@ -25,6 +27,8 @@
               files="Fetcher.java"/>
     <suppress checks="ParameterNumber"
               files="ConfigDef.java"/>
+    <suppress checks="ParameterNumber"
+              files="EosLogEntry.java"/>
```

Comment:
Now that we have increased the param limit in checkstyle, do we still need this?